### DISCUSSION TITLE
Domain created by antmedia suddenly inaccessible.

### DISCUSSION THREAD

**Discussion by SatlujNetwork** [ant-media/Ant-Media-Server#5244] [Q&A] [answered]
Hi, my domain is `ams-25567.antmedia.cloud:5543`. But it suddenly can not be access. it says:

`DNS_PROBE_FINISHED_NXDOMAIN`

but when i access it using server ip itworks: `serverIp:5080`

**Comment by mekya** [comment, marked as answer]
Hi @SatlujNetwork,

If the domains are not used for some time(7-10 days), it's being removed automatically.  @muratugureminoglu can give some more accurate information about that.  

Alternatively, you can get another domain and use it on your end. 

Cheers

**Comment by muratugureminoglu** [comment]
Hi @SatlujNetwork 

I apologize for the delayed response. As I checked, I did not encounter any DNS issues and I can access "https://ams-25567.antmedia.cloud:5443" without any problems. There might have been a temporary DNS issue. As Oguz suggested, if you run the "enable_ssl.sh" script again, it will create a new subdomain.

**Reply by SatlujNetwork** [reply]
Hi @mekya and @muratugureminoglu 
Thank you for your previous response. I appreciate your help.

The last time I posted this issue, our antmedia domain was inaccessible, but a few hours after posting the problem, it became accessible again.

-----

However, the problem has resurfaced after a two-week period of stability. I am currently unable to access the Antmedia domain once again. Could you please provide further assistance or suggestions to resolve this issue?
Any suggestions?

Note: The last time our uploader uploads video was a week ago, is this could be the issue?

**Reply by muratugureminoglu** [reply]
Hi @SatlujNetwork 

It seems like there is currently an issue related to our automation. I apologize for that. I have re-added your DNS record. In order to prevent this situation from happening again, we will review the automation once more.

**Reply by SatlujNetwork** [reply]
I can confirm it is now working.
Thank you for your quick action, I appreciate it!
